File: Patients/Models/Patient.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Patients.Models
{
    public class Patient
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }


        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Birthday { get; set; }
        public char Gender { get; set; }
    }
}

File: Patients/ClientApp/src/app/Models/Patient.ts
export class Patient {
  FirstName: string;
  LastName: string;
  Birthday: string;
  Gender: string;

  constructor(FirstName: string, LastName: string, Birthday: string, Gender: string) {
    this.FirstName = FirstName;
    this.LastName = LastName;
    this.Birthday = Birthday;
    this.Gender = Gender;
  }
}
